src/kvstorageclient/etcd_client.h

```diff
@@ -165,32 +165,38 @@ class EtcdClientImp : public KVStorageClient {
 
     /**
      * @brief ListWithLimitAndRevision
-     *        获取指定个数，指定revision，[startKey, endKey)的键值对
-     *
-     * @param[in] startKey 起始key
-     * @param[in] endKey 终止key, 不包含
-     * @param[in] limit 最多个数
-     * @param[in] revision 获取<=revision的key
-     * @param[out] values 所有键值对的value集合
-     * @param[out] lastKey 集合的最后一个key值
+     *        get key-value pairs between [startKey, endKey)
+     *        with specify number and revision
+     *
+     * @param[in] startKey start key
+     * @param[in] endKey end key, not included
+     * @param[in] limit max number
+     * @param[in] revision get the key <= revision
+     * @param[out] values the value vector of all the key-value pairs
+     * @param[out] lastKey the last key of the vector
      */
     virtual int ListWithLimitAndRevision(const std::string &startKey,
         const std::string &endKey, int64_t limit, int64_t revision,
         std::vector<std::string> *values, std::string *lastKey);
 
     /**
-     * @brief CampaignLeader 通过etcd竞选leader,如果成功则返回; 如果未竞选成功，
-     *      electionTimeoutMs>0的情况下会返回失败， electionTimeoutMs=0的情况
-     *      会一直block住，直到当选leader或者出现fatal error
-     *
-     * @param[in] pfx 竞选leader要写的key
-     * @param[in] leaderName leader名字，建议使用ip+port的形式表示唯一
-     * @param[in] sessionInterSec 用于创建带ttl的session, 如果client挂掉,
-     *                            esession超时后，etcd会把该leader注册的节点删除
-     * @param[in] electionTimeoutMs 该接口的超时时间，为0会一直block
-     * @param[out] leaderOid leader的objectId，记录在objectManager中
-     *
-     * @return EtcdErrCode::EtcdCampaignLeaderSuccess表示成功，其他表示失败
+     * @brief CampaignLeader leader campaign in etcd, success: return
```

Comment:
Leader campaign through etcd, return directly if the election is successful. Otherwise, if electionTimeoutMs>0, it will return failure; if electionTimeoutMs=0, it will block until become leader or fatal error appears.

Reply:
> Leader campaign through etcd, return directly if the election is successful. Otherwise, if electionTimeoutMs>0, it will return failure; if electionTimeoutMs=0, it will block until become leader or fatal error appears.

done